node/plugin/cmd/main.go

```diff
@@ -128,31 +128,31 @@ func pluginOps(ctx *cli.Context) {
 		RegisterNodeAtStart: false,
 	}
 	churnerClient := node.NewChurnerClient(config.ChurnerUrl, true, operator.Timeout, logger)
-	if config.Operation == "opt-in" {
+	if config.Operation == plugin.OperationOptIn {
 		log.Printf("Info: Operator with Operator Address: %x is opting in to EigenDA", sk.Address)
 		err = node.RegisterOperator(context.Background(), operator, tx, churnerClient, logger.With("component", "NodeOperator"))
 		if err != nil {
 			log.Printf("Error: failed to opt-in EigenDA Node Network for operator ID: %x, operator address: %x, error: %v", operatorID, sk.Address, err)
 			return
 		}
 		log.Printf("Info: successfully opt-in the EigenDA, for operator ID: %x, operator address: %x, socket: %s, and quorums: %v", operatorID, sk.Address, config.Socket, config.QuorumIDList)
-	} else if config.Operation == "opt-out" {
+	} else if config.Operation == plugin.OperationOptOut {
 		log.Printf("Info: Operator with Operator Address: %x and OperatorID: %x is opting out of EigenDA", sk.Address, operatorID)
 		err = node.DeregisterOperator(context.Background(), operator, keyPair, tx)
 		if err != nil {
 			log.Printf("Error: failed to opt-out EigenDA Node Network for operator ID: %x, operator address: %x, quorums: %v, error: %v", operatorID, sk.Address, config.QuorumIDList, err)
 			return
 		}
 		log.Printf("Info: successfully opt-out the EigenDA, for operator ID: %x, operator address: %x", operatorID, sk.Address)
-	} else if config.Operation == "update-socket" {
```

Comment:
oh we had this before but didn't add it to OperationFlag

Reply:
yep lol